addon/mixins/responds-to-print.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+// Triggers 'print' event and calls 'print' handler when the page is being printed.
+// Browser support: http://caniuse.com/#feat=matchmedia
+export default Ember.Mixin.create(Ember.Evented, {
+
+  print: Ember.$.noop,
+
+  didInsertElement: function () {
+    this._super();
+
+    this._RespondsToPrint_mediaQueryList = window.matchMedia('print');
```

Comment:
We can make the code more readable and make a minor performance improvement if we cache this global, constant reference in a shared closure variable.

e.g. `var mediaQueryList;` at top of file and then replace line 12 with `mediaQueryList = mediaQueryList || window.matchMedia('print');`